kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -3749,6 +3758,32 @@ messages:
       return;
    }
 
+   UserMoveInventoryItem(payload=$,target=$)
```

Comment:
Maybe 1 would be a better default value?  $ is never legal now that these are ints.  Also maybe change name to payload_index and target_index?  Function could also use a comment describing what it does and what parameters mean.

Reply:
Updated names and prepopulated value to 1.  Also added description to the function.